enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,259 @@
+4. A boot image is created using the CoreOS OVA.
+5. Once imported the installer can start building the private network for the nodes.
+6. The bootstrap and master nodes will be created on in the private network. These are called Power VS instances,
+and they get created using the CoreOS boot image.
+7. In order to expose this private network and provide load balancing, the IBM Cloud APIs will leverage the IBM
+Cloud [Direct Link][direct-link-website] offering. This will allow for both the ingress load balancer and the cluster
+nodes to talk over the private network on a high-speed link.
+8. Bootstrapping completes as normal.
+9. The workers will be added using the openshift/[cluster-\*machine-\*] controllers and operators based on the openshift
+MAPI model. A Power VS controller needs to be implemented and hooked into the
+[OpenShift Machine Operator][openshift-machine-operator].
+10. The cluster teardown can make use of the existing [go client for Power][power-go-client].
+
+### User Stories
+
+- A user of this installer would like to see a cluster running within an hour of kicking off the installer.
+- The expectation is that this is fully automated.
+
+### Implementation Details/Notes/Constraints
+
+#### CoreOS OVA Considerations
+There were two main implementation ideas for making the CoreOS OVA boot image available to the Virtual Server Instance.
+
+1. Import the Image from a Cloud Store
+This is the implementation we believe is best, since it works most similarly to other cloud environments. In this setup,
+the CoreOS build pipeline would publish an OVA image to an IBM Cloud Object Storage (COS) bucket. Then, we will use the
+regular IBM Cloud APIs to import the image to the instance. The drawback to this approach is the the current
+implementation is very slow. In its current state, it takes about 30 minutes to import an 120 Gb boot image. This is too
+long to meet our target of being able to install a whole cloud within the hour. One of the reasons this is slow is
+because we cannot do the image resizing after the copy, since it's writing directly to a sized disk. Once imported,
+creating the nodes is fast, since the other disks are flashed from the snapshot. For now, we asked the team to look into
+meeting the requirement that importing the image can take 5 minutes or less.
+
+2. Published CoreOS Images to the IBM Cloud Pre-Authorized Default Images
+This approach would involve outputting our CoreOS OVA images in a queue where they can be pre-loaded into the IBM Cloud
+so that they are available as soon as the Virtual Server Instance is available. This is significantly quicker than the
+method above but has several drawbacks. For one, these default images need to be pre-authorized in order for them to be
+added to the IBM Cloud. This is currently a manual process. Additionally, while we could build automation to solve this,
```

Comment:
it might be good to clarify that it's a manual process owned by Power VS (vs by a user)